package/run.sh

```diff
@@ -244,8 +245,8 @@ if [ -n "$CATTLE_CA_CHECKSUM" ]; then
         error "Please check if the correct certificate is configured at $CATTLE_SERVER/v3/settings/cacerts"
         exit 1
     else
-        mkdir -p /etc/kubernetes/ssl/certs
-        mv $temp /etc/kubernetes/ssl/certs/serverca
+        mkdir -p /etc/kubernetes/ssl/certs/$CATTLE_SERVER_HOSTNAME_WITH_PORT
+        mv $temp /etc/kubernetes/ssl/certs/$CATTLE_SERVER_HOSTNAME_WITH_PORT/ca.crt
```

Comment:
I think subdirectory in `/etc/kubernetes/ssl/certs/` won't work. In the default cert pool of golang https://github.com/golang/go/blob/29907b13db0455eded50263b4e37445045c82e6e/src/crypto/x509/root_unix.go#L64-L82, only the cert files in `/etc/kubernetes/ssl/certs/` can be loaded but not those in subdir.

Reply:
Updated. Now we are not touching the `serverca` file, but copying cacert to `/etc/docker/certs.d` additionally